src/abstractdataframe/abstractdataframe.jl

```diff
@@ -923,7 +923,7 @@ unique!(df)  # modifies df
 
 function without(df::AbstractDataFrame, icols::Vector{<:Integer})
     newcols = setdiff(1:ncol(df), icols)
-    df[newcols]
+    view(df, newcols)
```

Comment:
I'm not following this change. You are adding copies everywhere else, why a view here?

Reply:
This is a function that is not exported and is only used internally in `join` and `aggregate`. In both cases it is enough that we get a `SubDataFrame` there.